### DISCUSSION TITLE
Customizing GoBlog while in containers

### DISCUSSION THREAD

**Discussion by StarrWulfe** [jlelse/GoBlog#55] [Q&A]
Hey everyone, I've decided to make a go at using GoBlog (pun *definitely* intended 😉)

I've gotten some help with getting started from @kandr3s but he also noted his instance is installed on bare metal and isn't dockerized.

I'd like to hack away on the layout and css set up just to add a different font and move menus around in the template as well as figure out some of the plug in functionality. However the instructions in docs.goblog.app are geared towards those installing directly. 

Should I just mount `/original-assets/styles` and go about it that way or is there some other possibility by way of using the already mounted `/static` that can be done on the fly?

**Comment by jlelse** [comment]
I'm thinking about a good way to add custom CSS. It's on my todo list. Until then the best way is probably to create a fork, modify the original SCSS, compile it using `original-assets/build/buildStyles.sh` and then build your Docker image. Then regularly rebase your fork on the latest GoBlog commits.

**Reply by StarrWulfe** [reply]
Perhaps creating a system where a "child theme" could overwrite the default is the way to go; this is how Wordpress theming and CSS works, and even in Dockerized installs could find the needed CSS child layout in `/static` or a new mount.

A way of automatically kicking off `buildStyles.sh` would be needed to be done whenever a child file is done though... Is it possible?

**Comment by jlelse** [comment]
Update: There's a new plugin for custom CSS. I hope that is a first helpful step. Feel free to share your thoughts!

**Reply by StarrWulfe** [reply]
Thanks @jlelse  
Going to pull down the changes and see if I can’t do a little light customization here.
I have some other ideas I’d like to ask about in terms of improving GoBlog if it’s ok… Can I make separate issues for them?

https://starrwulfe.xyz/s/4c

**Reply by jlelse** [reply]
> Can I make separate issues for them?

Sure!
